src/util.ts

```diff
@@ -688,6 +689,18 @@ export const getIncreasedPriceFromExisting = (
   return getIncreasedPriceHex(convertPriceToDecimal(value), rate);
 };
 
+export const validateGasValues = (gasValues: GasValues) => {
+  Object.keys(gasValues).forEach((key) => {
+    const value = (gasValues as any)[key];
+    if (typeof value !== 'string' || !isHexString(value)) {
+      throw new TypeError(
+        `expected hex string for ${key} but received: ${value}`,
+      );
+    }
+  });
+  return true;
```

Comment:
Nit: May as well return nothing, if the expectation is that this will throw upon error. Otherwise it might look like we're throwing away the return value when we ignore it.